PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/bidding/data/bid/Prebid.java

```diff
@@ -124,6 +124,9 @@ public static JSONObject getJsonObjectForBidRequest(
         if (config.isOriginalAdUnit() && config.getAdFormats().size() > 1) {
             Utils.addValue(targeting, "includeformat", "true");
         }
+        Utils.addValue(targeting, "includewinners", PrebidMobile.getIncludeWinnersFlag());
```

Comment:
Let's add unit tests for these new properties to avoid regression in future.

@ValentinPostindustria please show the existing tests or files where the respective checks should be added

Reply:
I'll add the tests once I come back. What test should I use as an example? I have never made one test yet in my one year career 😂

Reply:
Add test into `org.prebid.mobile.rendering.bidding.data.bid.PrebidTest`. Don't forget to return it to the default value in `@After` because it's static.

Reply:
> I'll add the tests once I come back. What test should I use as an example? I have never made one test yet in my one year career 😂

Great! And don't worry - Valentin will help to make everything right.